fc_port.go

```diff
@@ -36,6 +36,17 @@ func (c *ClientIMPL) GetFCPorts(
 	err = c.readPaginatedData(func(offset int) (api.RespMeta, error) {
 		var page []FcPort
 		qp := getFCPortDefaultQueryParams(c)
+
+		var OEVersion string
+		softwareInstalled, err := c.GetSoftwareInstalled(ctx)
+		for _, software := range softwareInstalled {
+			if software.IsCluster == true {
+				OEVersion = software.BuildVersion
+			}
+		}
+		if OEVersion == "3.0.0.0" {
```

Comment:
This will not work in case the array gets patched with `3.x.x.` ? I would recommend just checking for the major version i.e. `3`
Maintain this in a constant